### PR TITLE
[sqlpp11] update to v0.59

### PR DESCRIPTION
**Pull request by yurybura**
This PR updates sqlpp11 library to v0.59 and adds features to sqlpp11-connector-mysql to support MySQL connector.

### CODE REVIEW

**Comment by JackBoosY** [comment]
Thanks for this PR! But I've fixed cpp-taskflow and fastrtps regressions, so please revert them.

**Review by JackBoosY — APPROVED** [review]

File: ports/sqlpp11/portfile.cmake [hunk 1]
```diff
@@ -16,12 +14,11 @@ vcpkg_configure_cmake(
...
+
 vcpkg_install_cmake()
 
 # Move CMake config files to the right place
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/Sqlpp11 TARGET_PATH share/sqlpp11)
-
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/Sqlpp11 TARGET_PATH share/sqlpp11)
 
 # Delete redundant and unnecessary directories
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug ${CURRENT_PACKAGES_DIR}/lib)
```
Comment (LilyWangL) [inline ports/sqlpp11/portfile.cmake:24]: Can you please use command `file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)` to handle copyright?

File: ports/sqlpp11/CONTROL [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Source: sqlpp11
-Version: 0.58-3
+Version: 0.59
```
Comment (LilyWangL) [inline ports/sqlpp11/CONTROL:2]: Can you please add the Homepage for this port in the CONTROL file?

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/sqlpp11/CONTROL (hunk 2 above)
Comment (yurybura) [reply ports/sqlpp11/CONTROL:2]: Added.

**Comment by LilyWangL** [comment]
Waiting for PR #11824 merged, this PR will fix `qscintilla` build failed on CI.

**Review by LilyWangL — APPROVED** [review]

**Pull request merged**